lxd/storage/drivers/utils.go

```diff
@@ -226,6 +227,36 @@ func createSparseFile(filePath string, sizeBytes int64) error {
 	return nil
 }
 
+// ensureVolumeBlockPath creates or resizes the raw block file for a volume.
+func ensureVolumeBlockPath(vol Volume, path string) error {
```

Comment:
@stgraber how about calling this `ensureVolumeBlockFile` as the comment suggests it only operates on files, but `ensureVolumeBlockPath` suggests it could also operate on devices on a path.

Reply:
fixed